docker/docker-py3-kms/Dockerfile

```diff
@@ -1,28 +1,27 @@
 # Switch to the target image
 FROM alpine:3.15
 
-ENV IP 0.0.0.0
+ARG BUILD_COMMIT=unknown
+ARG BUILD_BRANCH=unknown
+
+ENV IP ::
```

Comment:
Are we defaulting to IPv6 now?

Reply:
Yes, we do - as requested in #71 and I don't see any problems with it.